cockatrice/src/decklistmodel.cpp

```diff
@@ -300,8 +300,9 @@ QModelIndex DeckListModel::addCard(const QString &cardName, const QString &zoneN
 
 QModelIndex DeckListModel::nodeToIndex(AbstractDecklistNode *node) const
 {
-    if (node == root)
+    if (node == nullptr || node == root)
```

Comment:
I took a look and saw nothing that should mind it. It wants an empty list unless the list is defined already